packages/host/tests/helpers/mock-matrix.ts

```diff
@@ -53,6 +81,17 @@ export function setupMockMatrix(
         instantiate: false,
       },
     );
+    this.owner.register(
+      'service:matrix-mock-utils',
+      {
+        async load() {
+          return mockUtils;
+        },
+      },
+      {
+        instantiate: false,
+      },
+    );
```

Comment:
Maybe we can think of a more elegant way to do this?

Reply:
👍🏻 I finally [faced it](https://github.com/cardstack/boxel/pull/2116/commits/1a1ef30b08ef9645a4bc8027b00d85a1d4b4c51c), thanks